TFLite/src/common/Storage.py

```diff
@@ -0,0 +1,208 @@
+import os
+from queue import Queue
+from google.cloud import spanner
+
+# Module to store Graph data into database
+# Node object attributes are stored in 'Operators' table
+# Edge object attributes are stored in 'Tensors' table
+# Graph object attributes are stored in 'Models' table
+
+# The attribute names in each case must correspond to a column name in 
+# the database. If any attribute is added in the Class files, 
+# the Storage class will automatically try to add that attribute to the 
+# database the next time it is run with any model
+
+# If an attribute is added to the class file and not to the database,
+# this will throw and error unless explicitly changed
+
+# Spanner allows only 20000 mutations per commit, so _load_operators
+# and _load_tensors implement batching
+class Storage:
+
+    # Initialize spanner client with instance and database
+    def __init__(self, instance_id, database_id):
+        self.spanner_client = spanner.Client()
+        self.instance = self.spanner_client.instance(instance_id)
+        self.database = self.instance.database(database_id)
+
+    # Internal function to load graph metadata into the Models table
+    def _load_model(self, graph):
+        # To store the database column names and their values to be inserted
+        columns = list()
+        values = list()
+
+        # Adding Graph attributes and their values to columns and values
+        attrs = vars(graph)
+        for item in attrs.items():
+            if (item[0] != 'nodes' and item[0] != 'start_node_indices' 
```

Comment:
nit: Maybe add these to an 'exception_list'/'exclusion_list' and keep it in one place so that this if can be a presence check?

Reply:
Added in subsequent PR